app/api/log.py

```diff
@@ -41,4 +42,9 @@ def log(data):
         if not room:
             return False, "room not found"
 
-    log_event(data["type"], current_user, room=room, data={'message':data['message']})
\ No newline at end of file
+    reduced_data = data
+    if 'type' in data: del data['type']
```

Comment:
You are removing the values from `data`, not from `reduced_data`

Reply:
ohh sorry